11_install_agent/30_install_agent_suse.txt

```diff
@@ -16,7 +16,7 @@ zypper ar -n 'Rudder SLES repository' http://www.rudder-project.org/rpm-3.0/SLES
 
 ----
 
-zypper sa 'http://www.rudder-project.org/rpm-3.0/SLES_10_SP3/' Rudder
+zypper sa 'http://www.rudder-project.org/rpm-3.1/SLES_10_SP3/' Rudder
```

Comment:
Can you add a section about SLES12 here please ? 
zypper ar -n 'Rudder SLES 12 repository' http://www.rudder-project.org/rpm-3.1/SLES_12/ Rudder